docs/reference/api/docker_remote_api_v1.22.md

```diff
@@ -1133,6 +1136,7 @@ Attach to the container `id`
 
 Query Parameters:
 
+-   **detacheys** – Specify the escape key sequence used to detach a container.
```

Comment:
`Specify a custom key sequence for detaching a container. Format is ctrl-<value> where <value> is one of: a-z, @, ^, [, ``, or _`